client/src/helpers/Modal.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react'
+import ModalContext from '../contexts/ModalContext'
+
+
+export default () => {
+  const {
+    modalIsOpen,
+    setModalIsOpen,
+    headingText,
+    setHeadingText,
+    contentText,
+    setContentText
+  } = useContext(ModalContext)
+
+  if (!modalIsOpen) {
+    return <div />
+  }
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    setHeadingText(false)
+    setContentText("")
+    setModalIsOpen(false)
+  }
+
```

Comment:
Possibly move the conditional below the handle click.  That way it is closer to the other return statement.